### ISSUE TITLE
broken expander when "no_results_found"

### ISSUE THREAD

**Issue by photomedia** [eprints/irstats2#79]
90_irstats2.js has the following on line 252:

			var elparent = container.up( "div[class=ep_summary_box_body]" );

This breaks the EPrints::Box, because it removes the ..._inner class that is required.
This should be changed to:

			var elparent = container.up( "div[class=irstats2_summary_page_container]" );

This way, if there is an error message (eg: no stats), the expander is not broken.

**Comment by wfyson** [comment]
To try and keep the look and feel of the box the same, I'd suggest the following fix instead. Rather than change the element we update, we update the element in a manner that allows it to still be collapsed.

```
// for summary page
var elparent = container.up( "div[class=ep_summary_box_body]" );
if( elparent != null )
{       
    elparent.update( "<div id='" + elparent.id + "_inner'><p>" + msg + "</p></div>" );
    return;
}
```

**Comment by photomedia** [comment]
Have you tested your fix?  If it works and the expand/collapse button is working even when there is an error message (no stats), then it's fine with me :)

**Comment by wfyson** [comment]
Yeah, have tested and it works - will roll it into the codebase and update the bazaar as soon as possible!

**Comment by goetzk** [comment]
This code appears to be in place within `lib/static/javascript/auto/90_irstats2.js`, so it seems this has been done done.
